rules/manager.go

```diff
@@ -311,6 +314,20 @@ func (g *Group) SetEvaluationDuration(dur time.Duration) {
 	g.evaluationDuration = dur
 }
 
+// SetEvaluationTimestamp updates evaluationTimestamp to the Unix timestamp of when the rule was last evaluated.
+func (g *Group) SetEvaluationTimestamp(ts time.Time) {
+	g.mtx.Lock()
+	defer g.mtx.Unlock()
+	g.evaluationTimestamp = ts
+}
+
+// GetEvaluationTimestamp returns the time the evaluation took place.
+func (g *Group) GetEvaluationTimestamp() string {
+	g.mtx.Lock()
+	defer g.mtx.Unlock()
+	return g.evaluationTimestamp.UTC().Format("15:04:06 MST")
```

Comment:
We should also not expose anything other than UTC.

Reply:
It's UTC here, the MST is just there because that's what the Go time formatter expects as a reference time zone.

Reply:
I've fixed everything else based on your feedback, but left this part the for the time being while a decision is made.

Reply:
What if we display the time since the last evaluation as it is done for scrape targets?

Reply:
Yeah, we could do that too and it would seem more consistent with the targets page. Then it would also solve our template function problem as we can just use `humanizeDuration` again :)

@wbh1 What do you think, is there a need for the specific timestamp or would a duration since the last evaluation also be sufficient?

Reply:
That works for me.

Is everyone okay with the column header "Last Evaluation" and using the "\<seconds\> ago" format?

(Like how /targets uses: ```{{if .LastScrape.IsZero}}Never{{else}}{{since .LastScrape}} ago{{end}}``` )

Reply:
This has been updated (and it looks nicer, too. Thanks, @simonpasquier ! 😄)